packages/preview/biz-report/0.1.0/README.md

```diff
@@ -0,0 +1,28 @@
+# Business Report 
+
+This is a business report template in Typst
+
+![Screenshot of welcome page](thumbnail.png)
+
+## Location
+
+The source for this guide is held in an Github repository. Clone it from: 
+
+https://github.com/garethwebber/business-report
+
+## Editing
+
+It is recommended to us Visual Studio code to edit the document. It can manage the connection to
+the git repository and if you install _Tintmist Typst_ as a module you will get auto complete, syntax highlighting, and best of all, live preview. https://marketplace.visualstudio.com/items?itemName=myriad-dreamin.tinymist
+
+## Dependencies
+
+Typst must of course be installed (https://github.com/typst/typst/releases/download/v0.14.0/typst-x86_64-pc-windows-msvc.zip). The template makes use of two fonts which must be installed on the local machine:
```

Comment:
I think linking to `https://typst.app/open-source#download` would be more relevant. It may be worth mentioning that using the web app is an option too. Also, you only listed one font, not two.